doc/users/next_whats_new/polar_errorbar_caps.rst

```diff
@@ -0,0 +1,7 @@
+Fixed errorbars in polar plots
+------------------------------
+Caps and error lines are now drawn with respect to polar coordinates,
+when plotting errorbars on polar plots.
+
+.. figure:: /gallery/pie_and_polar_charts/images/sphx_glr_polar_error_caps_001.png
```

Comment:
If this works, great!  (not sure it will, but my ignorance of sphinx is relatively vast)

Reply:
I just mimicked it from [this](https://github.com/matplotlib/matplotlib/blob/main/doc/users/prev_whats_new/whats_new_3.5.0.rst#image-interpolation-now-possible-at-rgba-stage) what's new message, but it does not seem to work properly in the built documentation, so I am not sure it is worth the try.